optuna/integration/sklearn.py

```diff
@@ -43,33 +43,30 @@
 from optuna import trial as trial_module  # NOQA
 from optuna import type_checking  # NOQA
 
-if type_checking.TYPE_CHECKING:
-    import pandas as pd  # NOQA
-    from scipy.sparse import spmatrix  # NOQA
-    from typing import Any  # NOQA
-    from typing import Callable  # NOQA
-    from typing import Dict  # NOQA
-    from typing import Iterable  # NOQA
-    from typing import List  # NOQA
-    from typing import Mapping  # NOQA
-    from typing import Optional  # NOQA
-    from typing import Union  # NOQA
-
-    ArrayLikeType = Union[List, np.ndarray, pd.Series, spmatrix]
-    OneDimArrayLikeType = Union[List[float], np.ndarray, pd.Series]
-    TwoDimArrayLikeType = Union[List[List[float]], np.ndarray, pd.DataFrame, spmatrix]
-    IterableType = Union[List, pd.DataFrame, np.ndarray, pd.Series, spmatrix, None]
-    IndexableType = Union[Iterable, None]
+# if type_checking.TYPE_CHECKING:
+import pandas as pd  # NOQA
+from scipy.sparse import spmatrix  # NOQA
+from typing import Any  # NOQA
+from typing import Callable  # NOQA
+from typing import Dict  # NOQA
+from typing import Iterable  # NOQA
+from typing import List  # NOQA
+from typing import Mapping  # NOQA
+from typing import Optional  # NOQA
+from typing import Union  # NOQA
```

Comment:
Could you move these imports to the top? Standard library imports should come first, then third party, lastly optuna internal. Conditionals are exceptions though.